### ISSUE TITLE
(MAC) instantly closes

### ISSUE THREAD

**Issue by Mahdiarfish** [cheat-engine/cheat-engine#1658]
CE 7.2 mac instantly closes when I open it. It doesn't pop up with an error so IDK what the problem is.

**Comment by Mahdiarfish** [comment]
It also says with a finder icon popup, "The application cheat engine 7.2 cannot be opened."

**Comment by cheat-engine** [comment]
try launching it from the terminal using sudo

and if you're on an oldet mac build then replacr Contents/MacOS/liblua53.dylib with liblua53.dylib.working

**Comment by Mahdiarfish** [comment]
I'm so sorry, but how do u launch it using sudo? I'm not good at this stuff.

**Comment by Mahdiarfish** [comment]
Also is 2014 considered old. I'm on Mojave if that matters.

**Comment by Mahdiarfish** [comment]
For the sudo thing, it says this :[FORMS.PP] ExceptionOccurred 
  Sender=EAccessViolation
  Exception=Access violation
  Stack trace:
  $0000000103F08D6B
Exception at 0000000103F08D6B: EAccessViolation:
Access violation.

**Comment by Mahdiarfish** [comment]
Oh ok, I found the .dylib. How do I replace it, do I move the other one to trash?

**Comment by cheat-engine** [comment]
yes. move the old on to trash or rename it and then rename liblua53.dylib.working to liblua53.dylib

**Comment by Mahdiarfish** [comment]
Still doesn't work. I moved the first one to the trash and did what you said.

**Comment by Mahdiarfish** [comment]
That happens when i go to /Users/downloads/cheat engine/macos and i clicked on the top one.

**Comment by Mahdiarfish** [comment]
Hello?

**Comment by Mahdiarfish** [comment]
tis might be more detailed. you know how when you open something the icon bobs up and down? so it goes up once then stops and closes, i dont even see it open.

**Comment by cheat-engine** [comment]
Perhaps this version works better: [CheatEngine73 for MacOS.zip](https://github.com/cheat-engine/cheat-engine/files/6993469/CheatEngine73.for.MacOS.zip)

**Comment by Mahdiarfish** [comment]
still didnt work, same issue though.

**Comment by Mahdiarfish** [comment]
maybe an older version might work better.

**Comment by Brxdlee** [comment]
I am also having the same issue, with The latest download of cheat engine on a Mid-2011 iMac with a 2.5 GHz Intel Core i5 Processor. I cannot open the app after opening the CheatEngine73.zip file I've tried many commands like codesign command in terminal and it didn't help. I've also tried removing the dylib53.working file and renaming the one without working to working and it had absolutely no effect on the app instantly crashing on opening.

**Comment by cheat-engine** [comment]
you use sudo to launch it?

**Comment by Brxdlee** [comment]
No, I did not use sudo I just opened it manually from the file it was stored in when downloaded. The reason I am even here is that I really enjoyed this application and all the hard work, as well as the in-depth features it offers. It was nice being able to use this application because it taught me many things and how values are stored and named and hex values, etc. I have used this app since 2018 it comes in clutch when I cannot beat a single-player game (AFTER HOURS) it's so fun and I would definitely pay and watch ads to support it. But I choose not because of the issues I'm encountering. I previously had the 6.x Mac v5 Beta version (No issues, Crashes, Bugs) until I deleted it today because I thought a newer version would have more features and run smoother but I regret it. At least I know now that I was doing something illegal by using an app that is no longer free which I actually ended up being okay about deleting because I do not support the use of someones else's hard work without permission or legal use of it. So I hope you can help me out on this issue or I will just forget about cheat engine and all the fun memories that I had. Thanks in advance.

**Comment by cheat-engine** [comment]
it should still work even if you don't pay 
try running it with sudo 
and see if there's any system logs that may say why it won't load.  
The current version is 64-bit only.  Is your system capable of 64-bit?

**Comment by Brxdlee** [comment]
Yes, it is it has a 2.5 GHz Intel Core i5 and i5 does support 64-bit. Actually, nevermind I just opended it and it asked me to pay or use trial i'm so happy the sudo command worked!!!! All I did was typed this command `sudo /Applications/CheatEngine73.app/Contents/MacOS/cheatengine-x86_64 ` Thanks a lot the UI looks way better than the 2012 one that I had previously.

**Comment by AnzhiZhang** [comment]
Use `sudo /Applications/CheatEngine752.app/Contents/MacOS/cheatengine-x86_64` on M4 Mac. It instantly closed.

Edit: Deleted `.xml` file in the user dir, seems like I pressed the wrong button at the first launch.
